app/views/course/leaderboards/groups.html.slim

```diff
@@ -0,0 +1,37 @@
+= page_header format_inline_text(@leaderboard_settings.group_leaderboard_title || t('.header'))
+
+= render partial: 'tabs'
+
+div.col-sm-12.col-md-6
+  div.leaderboard-header
+    h4 = t('.experience_points_header')
+  table.table.leaderboard-points
+    tbody
+      - course_groups_points = @course_groups.ordered_by_experience_points.take(display_user_count)
+      - course_groups_points.each.with_index(1) do |group, index|
+        = content_tag_for(:tr, group, class: ['course-group'])
+          td.rank
+            h3 = index
+          td.group-name
+            h3 = format_inline_text(group.name)
+          td.group-profile
+            - group.course_users.includes(:user, :course).students.take(5).each do |course_user|
+              = link_to_course_user(course_user) do
+                = display_user_image(course_user.user)
+
+div.col-sm-12.col-md-6
+  div.leaderboard-header
+    h4 = t('.achievement_count_header')
+  table.table.leaderboard-achievement
+    tbody
+      - groups_count = @course_groups.ordered_by_average_achievement_count.take(display_user_count)
+      - groups_count.each.with_index(1) do |group, index|
```

Comment:
Implement a partial named `_group`, then just `render @course_groups` ?  Save some code and don't need to duplicate the same view logic twice.